app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -26,6 +26,17 @@ description: |
     '/EFI/opensuse/grubx64.efi',
     '/EFI/Microsoft/boot/bootmgfw.efi'
   ]
+  config_paths = [
+    '/EFI/fedora/grub.cfg',
```

Comment:
When I spin up my VM with Libvirt, I get the Grub2 default file, but when I select the `Loading GRUB2 config from ESP` item, I always end up with the error `no such device /EFI/fedora/grub.cfg`.

I tried to put the dir structure like this:
```
/var/lib/tftpboot/EFI/...
/var/lib/tftpboot/grub2/EFI/...
/EFI/...
```
And nothing works. The `/EFI/fedora/` path is also missing from the net install ISO for Fedora. What am I doing wrong?

Reply:
Just to be sure: Changes in this PR affect local boot from disk via network booted GRUB2 binary of already provisioned systems.

If there is no system installed at all and you get the default `grub2/grub2.cfg` then local boot will always fail - because there is nothing to boot from disk.

Reply:
Oooh, that makes sense. I knew I was doing something wrong.